api/service.py

```
@@ -486,7 +538,7 @@ async def metrics(username: str = Depends(get_current_username)):
 
 
 @app.post("/api/v1/telemetry", response_model=AnomalyResponse, status_code=status.HTTP_200_OK)
-async def submit_telemetry(telemetry: TelemetryInput, current_user: User = Depends(require_operator)):
+async def submit_telemetry(telemetry: TelemetryInput, request: Request, api_key: APIKey = Depends(get_api_key)):
```

Ruff (0.14.10)

Undefined name `APIKey`

(F821)